nipoppy_cli/nipoppy/cli/parser.py

```diff
@@ -306,6 +318,7 @@ def get_global_parser(
         add_help=False,
     )
     add_arg_help(global_parser)
+    add_arg_version(global_parser)
```

Comment:
We might want to add the `--version` flag to the subparsers as well.

See https://github.com/nipoppy/nipoppy/blob/eb1dfac51c180ad27c27e9885e1bab84e4db30e6/nipoppy_cli/nipoppy/cli/parser.py#L337-L342

Reply:
either way is fine with me

if the whole team is in favor then let's do it

Reply:
I would lean towards **not** having the `--version` flag for subparsers, since if the user wants to check the version then they shouldn't be using a subcommand. Both the main parser and the subparsers have `--help` but its behaviour is different, otherwise all the other global options really only apply to runtime behaviour. The `git` CLI doesn't have `--version` for subcommands either